### DISCUSSION TITLE
Live stream black screen on OBS

### DISCUSSION THREAD

**Discussion by NotTheRealTWCH** [f1multiviewer/issue-tracker#146] [Q&A] [answered]
When using OBS to capture live streams, the video stream is completely black, but you can see the video controls and sync button. Replays work completely fine. Could this be something to do with the DRM content protection? Is there a possible fix for this?

**Comment by LapsTimeOFF** [comment]
This is because of the DRM. Won't be fixed.

**Comment by f1multiviewer** [comment, marked as answer]
Nope, this is by design. Re-streaming F1TV has brought DRM upon us, and I'll do everything in my power to prevent people from re-streaming F1MV. Please do not use this app.
